### PR TITLE
feat: Allow console access.

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
This will allow access to courses and coupons if the user is in the group ecomm_console_client

### CODE REVIEW

File: ecommerce/courses/views.py [hunk 1]
```diff
@@ -9,12 +9,11 @@
 from django.core.management import call_command
 from django.http import Http404, HttpResponse
 from django.views.generic import TemplateView, View
+from ecommerce_extensions.core.mixins import ConsoleAccessMixin as StaffOnlyMixin
```
Comment (mariajgrimaldi) [inline ecommerce/courses/views.py:12]: easy but a bit misleading

**Comment by mariajgrimaldi** [comment]
how are you creating the group?

**Comment by andrey-canon** [comment]
> How are you creating the group?

admin, since I thought how much time I spend doing a data migration vs create that on admin

File: ecommerce/courses/views.py (hunk 1 above)
Comment (andrey-canon) [reply ecommerce/courses/views.py:12]: feel free to add a suggestion

**Review by mariajgrimaldi — APPROVED** [review]

**Comment by mariajgrimaldi** [comment]
Is this in stage already? at least to test the access to courses

**Review by magajh — APPROVED** [review]

**Comment by andrey-canon** [comment]
> Is this in stage already? at least to test the access to courses

working on stage

**Pull request merged**